lib/src/value_accessors/control_value_accessor.dart

```diff
@@ -91,3 +109,33 @@ abstract class ControlValueAccessor<ModelDataType, ViewDataType> {
     }
   }
 }
+
+class _WrapperValueAccessor<ModelDataType, ViewDataType>
+    extends ControlValueAccessor<ModelDataType, ViewDataType> {
+  final _ModelToViewValueCallback<ModelDataType, ViewDataType>?
+      _modelToViewValue;
+  final _ViewToModelValueCallback<ModelDataType, ViewDataType>?
+      _valueToModelValue;
+
+  _WrapperValueAccessor({
+    _ModelToViewValueCallback<ModelDataType, ViewDataType>? modelToViewValue,
+    _ViewToModelValueCallback<ModelDataType, ViewDataType>? valueToModelValue,
+  })  : _modelToViewValue = modelToViewValue,
+        _valueToModelValue = valueToModelValue;
+
+  @override
+  ViewDataType? modelToViewValue(ModelDataType? modelValue) {
+    if (_modelToViewValue != null && modelValue != null) {
+      return _modelToViewValue!.call(modelValue);
+    }
+    return null;
+  }
+
+  @override
+  ModelDataType? viewToModelValue(ViewDataType? viewValue) {
+    if (_valueToModelValue != null) {
+      return _valueToModelValue?.call(viewValue);
+    }
+    return control?.value;
```

Comment:
What if the *viewValue* argument is **null**? Won't you like to set **null** to the model FormControl value?

Reply:
Right, i will make it required  and let the implementation for the developer